support/build.gradle

```diff
@@ -37,17 +37,9 @@ if (JavaVersion.current().isJava8Compatible()) {
     }
 }
 
-// todo: will update this lib to sampingantech jcenter
-publish {
-    userOrg = 'nikart'
-    groupId = 'com.github.nikartm'
-    artifactId = 'image-support'
-    publishVersion = '1.1.0'
-    desc = 'Library to add ImageView (ImageBadgeView) with a badge like notification count'
-    licences = ['Apache-2.0']
-    uploadName='ImageBadgeView'
-    website = 'https://github.com/nikartm/Image-Support'
-}
+group = "com.github.sampingantech"
+version = android.defaultConfig.versionName
```

Comment:
to use same versionName, 

this also required to changes: https://github.com/sampingantech/Image-Support/blob/ebb11c0db26ba866be21c17107dd2be40adcabc8/support/build.gradle#L11

`versionName "1.0"` to `versionName "1.1.3"`

Reply:
well this is not required, due to this https://github.com/sampingantech/Image-Support/pull/8#discussion_r556382673